plexpy/webserve.py

```diff
@@ -1230,24 +1247,25 @@ def get_notification_log(self, **kwargs):
         return json.dumps(notifications)
 
     @cherrypy.expose
+    @cherrypy.tools.json_out()
     @requireAuth(member_of("admin"))
     @addtoapi()
     def clearNotifyLogs(self, **kwargs):
         data_factory = datafactory.DataFactory()
         result = data_factory.delete_notification_log()
+        result = result if result else 'no data received'
 
-        if result:
-            cherrypy.response.headers['Content-type'] = 'application/json'
-            return json.dumps({'message': result})
-        else:
-            cherrypy.response.headers['Content-type'] = 'application/json'
-            return json.dumps({'message': 'no data received'})
+        return {'message': result}
 
     @cherrypy.expose
     @requireAuth(member_of("admin"))
     def clearLogs(self):
-        plexpy.LOG_LIST = []
-        logger.info(u"Web logs cleared")
+        try:
+            open(os.path.join(plexpy.CONFIG.LOG_DIR, 'plexpy.log'), 'w').close()
```

Comment:
I liked the old way where clearing the logs would only clear it from the web UI, and not actually clear the file.

There have been times when I've had to help someone and they said they cleared the logs (which only cleared them from the web UI), but thankfully the log file was still complete.